TicketingSystem/src/cassa/Ticket.java

```diff
@@ -0,0 +1,85 @@
+package cassa;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ticket {
+    protected TicketType type;
+    protected Status status;
+    protected int pieces;
+    protected static List<Ticket> seatTicketList;
+    protected static List<Ticket> vipTicketList;
+    protected static List<Ticket> standTicketList;
```

Comment:
The `Ticket` class has a `TicketList` - it can work, but it violates some principles, because:
- you are putting different layers of an application (repository + model) in the same class
- you are violating the single responsibility principle

It is better to put different behaviours in different classes:
- first class: `Ticket`: what properties does a ticket have
- second class: `TicketRepository, which stores all the tickets